classes/Url.php

```diff
@@ -190,4 +190,52 @@ public function generate($encode = "html", $overrideVars = []) {
         }
     }
 
+    /**
+     * Generate a URL for the social image with the given header and subheader.
+     *
+     * @param string $header The header text for the social image.
+     * @param string $subheader The subheader text for the social image.
+     * @param string $parliament The parliament to use (default is "uk").
+     * @return string The URL to the generated social image.
+     */
+    public static function generateSocialImageUrl(string $header, string $subheader = "", string $parliament = "uk"): string {
+
+
+        function chamber_to_parliament($chamber) {
+            switch ($chamber) {
+                case 'house-of-commons':
```

Comment:
these also needs to handle 'uk-commons', 'uk-lords' and 'wales' otherwise you get an "Invalid parliament" error from the opengraph image generator. (or the open graph generator needs to accept those but that would mean more images).